lib/assessment.md

```diff
@@ -22,14 +22,27 @@ Using this format will make it easier for us to grade you, so that we can get ev
 
 When we say that the web is a service, what does that mean? Explain the interaction in terms of clients and servers.
 <!-- your answer starts here -->
-
+The we is a service, provide through an electronic networking backbone
+known as the internet -->
 <!-- your answer ends here -->
 
 ## Question 2
 
 Between the comments below, write HTML to create a basic web page (using the format described in the instructions). The page should have a title in the head and an unordered list (with three items) inside the body.
-<!-- your answer starts here -->
-
+<!-- your answer start here -->
+<html>
+<head>
+  <title>HTML AND CSS DIAGNOSTICS</title>
+</head>
```

Comment:
Watch your indenting - `head` and `body` are sibling elements, so they should be indented to the same depth.